controllers/providers/pg/applier.go

```diff
@@ -588,11 +588,21 @@ func (p *applier) genPGDataSourceSpec() (*crunchyv1beta1.DataSource, error) {
 	return pgDataSource, nil
 }
 
-func getPGRestoreOptions(dataSource everestv1alpha1.DataSource, backupBaseName string) ([]string, error) {
+func getPGRestoreOptions(
+	dataSource everestv1alpha1.DataSource,
+	backupStorage *everestv1alpha1.BackupStorage,
+	backupBaseName, repoName string) ([]string, error) {
 	options := []string{
 		"--set=" + backupBaseName,
 	}
 
+	if pointer.Get(backupStorage.Spec.ForcePathStyle) {
+		options = append(options, "--"+fmt.Sprintf(pgBackRestStorageForcePathTmpl, repoName)+"="+pgBackRestStoragePathStyle)
+	}
+	if !pointer.Get(backupStorage.Spec.VerifyTLS) {
+		options = append(options, "--no-"+fmt.Sprintf(pgBackRestStorageVerifyTmpl, repoName))
```

Comment:
Are you sure this works? I couldn't find this in the PGBackrest docs...
I could only find `repo1-storage-verify-tls=n`
https://pgbackrest.org/configuration.html#section-repository/option-repo-storage-verify-tls

Reply:
I wasn't able to get the anchor tag for this, but you'll find it under this section: https://pgbackrest.org/command.html#command-restore

<img width="1016" alt="Screenshot 2024-05-08 at 6 36 33 PM" src="https://github.com/percona/everest-operator/assets/30319228/c729148f-4424-4965-98e4-7b29c878ad1b">